Makefile

```diff
@@ -2,11 +2,11 @@
 .DEFAULT_GOAL := test
 
 PHPUNIT =  ./vendor/bin/phpunit -c ./phpunit.xml
-PHPDBG =  phpdbg -qrr ./vendor/bin/phpunit -c ./phpunit.xml
-PHPSTAN  = ./vendor/bin/phpstan
+PHPSTAN  = ./vendor/bin/phpstan --no-progress
```

Comment:
sometimes --no-progress is in here sometimes in the make target, chose one way to do it

Reply:
This is one place for no progress and applies to all